py3-jupyterhub-tmpauthenticator.yaml

```diff
@@ -27,10 +34,44 @@ pipeline:
       repository: https://github.com/jupyterhub/tmpauthenticator
       tag: v${{package.version}}
 
-  - name: Python Build
-    uses: python/build-wheel
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+      runtime:
+        - py${{range.key}}-jupyterhub
```

Comment:
pip shows that it has no runtime dependencies. 

```bash
$ uv pip show jupyterhub-tmpauthenticator
Name: jupyterhub-tmpauthenticator
Version: 1.0.0
Location: /Users/anurag.kumar/projects/uv-project/.venv/lib/python3.11/site-packages
Requires:
Required-by:
```

Reply:
Same here:

```
2025/01/06 12:58:49 WARN Traceback (most recent call last): uses=python/import
2025/01/06 12:58:49 WARN   File "<string>", line 1, in <module> uses=python/import
2025/01/06 12:58:49 WARN     import tmpauthenticator uses=python/import
2025/01/06 12:58:49 WARN   File "/usr/lib/python3.13/site-packages/tmpauthenticator/__init__.py", line 3, in <module> uses=python/import
2025/01/06 12:58:49 WARN     from jupyterhub.auth import Authenticator uses=python/import
2025/01/06 12:58:49 WARN ModuleNotFoundError: No module named 'jupyterhub' uses=python/import
```